src/main/resources/defaults/data/Banners.json

```diff
@@ -34,49 +34,33 @@
 		"gachaType": 301,
 		"scheduleId": 903,
 		"bannerType": "EVENT",
-		"prefabPath": "GachaShowPanel_A097",
-		"titlePath": "UI_GACHA_SHOW_PANEL_A097_TITLE",
+		"prefabPath": "GachaShowPanel_A103",
+		"titlePath": "UI_GACHA_SHOW_PANEL_A081_TITLE",
 		"costItemId": 223,
 		"beginTime": 0,
 		"endTime": 1924992000,
 		"sortId": 9998,
-		"rateUpItems4": [1072, 1065, 1053],
-		"rateUpItems5": [1071],
+		"rateUpItems1": [1073],
+		"rateUpItems2": [1073]
```

Comment:
`rateUpItems1` and `rateUpItems2` were deprecated 6 months ago. Additionally, you've set the featured 4-star pool to be the banner 5-star?